en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,202 @@
+
+The content of the box with the *PM02 V3* power module is shown below.
+This includes a pinout guide and power module instructions.
+
+![Durandal Box](../../assets/flight_controller/durandal/durandal_unboxing_1.jpg)
+
+The unpacked components, including cables, are:
+
+![Durandal - Fully Unboxed](../../assets/flight_controller/durandal/durandal_unboxing_2.jpg)
+
+
+## Wiring Chart Overview
+
+The image below shows how to connect the most important sensors and peripherals (except the motor and servo outputs).
+We'll go through each of these in detail in the following sections.
+
+![Durandal Wiring Overview](../../assets/flight_controller/durandal/durandal_wiring_overview.jpg)
+
+> **Tip** More information about available ports can be found here: [Durandal > Pinouts](../flight_controller/durandal.md#pinouts).
+
+
+## Mount and Orient Controller
+
+*Durandal* should be mounted on the frame positioned as close to your vehicle’s center of gravity as possible, oriented top-side up with the arrow pointing towards the front of the vehicle.
+
+![Mounting/Orientation](../../assets/flight_controller/durandal/orientation.jpg)
+
+If the controller cannot be mounted in the recommended/default orientation (e.g. due to space constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
+
+> **Tip** The board has internal vibration-isolation.
+  Do not use vibration-isolation foam to mount the controller (double sided tape is normally sufficient).
+
+
+## GPS + Compass + Buzzer + Safety Switch + LED
+
+Durandal is designed to work well with the *Pixhawk 4 GPS module*, which has an integrated compass, safety switch, buzzer and LED.
+It connects directly to the [GPS port](../flight_controller/durandal.md#gps) using the 10 pin cable.
+
+<!-- Confirm how compass delivery works - separate or in the package? -->
+
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
+
+![Connect compass/GPS to Durandal](../../assets/flight_controller/durandal/connection_gps_compass.jpg) 
+
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle).
+  To disable the safety press and hold the safety switch for 1 second.
+  You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
+
+
+
+## Power
+
+You can use a power module or power distribution board to power motors/servos and measure power consumption.
+The recommended power modules are shown below.
+
+### PM02 v3 Power Module {#pm02_v3}
```

Comment:
@bkueng OK, I have rewritten this section to include both PM02 and PM07. The PM07 section mostly refers to the Pixhawk 4 docs, which cover the behaviour in detail. The specs for each board are at the end of the section.
This does look odd, but it is intentional. At some point in near future I will create a power module section and most of this info will go into that section rather than living here.

One further confirmation. You said that power distribution is just wiring. Is there any guidance on that wiring? Or is it just a case that you wire all the ESCs in parallel to the battery, and choose an esc that can take the voltage level for the batteries?

Reply:
> Or is it just a case that you wire all the ESCs in parallel to the battery, and choose an esc that can take the voltage level for the batteries?

Exactly.